miner/proposal_builder.go

```diff
@@ -566,6 +567,157 @@ func (pb *ProposalBuilder) initSignerData(ss *signerSession, lid types.LayerID)
 	return nil
 }
 
+func (pb *ProposalBuilder) initSignerDataFor(ctx context.Context,
+	ss *signerSession,
+	lid types.LayerID,
+	nodeID types.NodeID,
+) error {
```

Comment:
It's confusing to pass `signerSession` here. There is no _signer_. How about:
```suggestion
func (pb *ProposalBuilder) initSignerDataFor(
	ctx context.Context,
	session *session,
	lid types.LayerID,
	nodeID types.NodeID,
) error {
```

Btw,  this function could be reused in the `initSignerData` to avoid repetition, wdyt?

Reply:
adjusted and reused :+1: